modules/gdscript/gdscript_editor.cpp

```diff
@@ -842,6 +842,20 @@ static String _make_arguments_hint(const GDScriptParser::FunctionNode *p_functio
 		}
 	}
 
+	if (p_function->is_vararg()) {
+		if (!p_function->parameters.is_empty()) {
+			arghint += ", ";
+		}
+		if (p_arg_idx >= p_function->parameters.size()) {
+			arghint += String::chr(0xFFFF);
+		}
+		const GDScriptParser::ParameterNode *rest_param = p_function->rest_parameter;
+		arghint += "..." + rest_param->identifier->name + ": " + rest_param->get_datatype().to_string();
```

Comment:
If the vararg parameter has an empty array as implicit default value we should show that in the call hint IMO.

Reply:
I disagree. The rest parameter does not have a default value in the normal sense. You can't specify or change the value. The rest parameter is a list of extra arguments passed; if there are none, the list is empty, that's all. So I think it would be confusing to show the "default" value in the call hint, given that you can't write it in code either.

Reply:
You are right, I didn't take the context of arg hints into account when writing this, since they are relevant for using a function not implementing it, it would get confusing. My bad.